batch_job/src/batch_job_mesos.c

```diff
@@ -82,7 +82,8 @@ static void start_mesos_scheduler(struct batch_queue *q)
 		ld_preload_str = string_format("LD_PRELOAD=%s", mesos_preload);
 	} 
 	// TODO: Environment variables set for launching Mesos, this would not work outside ND. Ask user to specify the environments?
-	char *envs[] = {"CCTOOLS=/afs/crc.nd.edu/user/c/czheng2/cctools", ld_preload_str, NULL};
+	char *cctools_path = string_format("CCTOOLS=%s", getenv("CCTOOLS"));
+	char *envs[] = {cctools_path, ld_preload_str, NULL};
 
```

Comment:
Er, that's not quite what I meant.  We don't generally require users to set the environment CCTOOLS.  Instead, you could use the macro CCTOOLS_INSTALL_DIR which is substituted at compile time.  Or, you could assume that makeflow (etc) is in the users PATH variable.